test/Bitfield.test.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable strict */
+const { Permissions } = require('../src');
+// You see, i understand this is supposed to be tests
+// for bitfields specifically but bitfields are abstract
+// and require more implementation before they can be used
+// so we're gonna use Permissions, okay? okay.
+
+describe('Bitfield unit tests', () => {
+  test('Bitfield#any', () => {
+    const bf = new Permissions([Permissions.FLAGS.SEND_MESSAGES, Permissions.FLAGS.VIEW_CHANNEL]);
+    expect(bf.any(Permissions.FLAGS.SEND_MESSAGES)).toStrictEqual(true);
+    expect(bf.any(Permissions.FLAGS.ADMINISTRATOR)).toStrictEqual(false);
+  });
+
+  test('Bitfield#equals', () => {
+    const bf = new Permissions([Permissions.FLAGS.SEND_MESSAGES, Permissions.FLAGS.VIEW_CHANNEL]);
+    // Redundant?
+    expect(
+      bf.equals(new Permissions([Permissions.FLAGS.SEND_MESSAGES, Permissions.FLAGS.VIEW_CHANNEL]).bitfield),
+    ).toStrictEqual(true);
```

Comment:
see comment

Reply:
Probably, but gotta get that test coverage up am I rite